atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClient.java

```diff
@@ -120,31 +134,40 @@ private int getApproximateWriteByteCount(Map<ByteBuffer, Map<String, List<Mutati
     public CqlResult execute_cql3_query(ByteBuffer query, Compression compression, ConsistencyLevel consistency)
             throws InvalidRequestException, UnavailableException, TimedOutException, SchemaDisagreementException,
             TException {
-
-        // This could be used for updating the schema mutation lock,
-        //  "UPDATE "%s" SET value = %s WHERE key = %s AND column1 = %s AND column2 = -1;"
-
-        // or to read the timestamp bound
-        // SELECT %s, %s FROM %s WHERE key=%s;
-
-        // Or for sweep?
-        // Should we consider all of these reads when recording metrics?
-        return checkLimitAndCall(() -> super.execute_cql3_query(query, compression, consistency));
+        CqlResult cqlResult = checkLimitAndCall(() -> super.execute_cql3_query(query, compression, consistency));
+        try {
+            recordBytesRead(SerializationUtils.serialize(cqlResult).length);
```

Comment:
feels like we want to do something smarter here than serializing the whole thing...

Reply:
Done.